### DISCUSSION TITLE
Things/proxies we need to add

### DISCUSSION THREAD

**Discussion by YesTheProxy** [theshmunguss1/RamOS#7] [Ideas]
If you want to add something put it in here and we'll force @DJshelfmushroom to do it

**Comment by YesTheProxy** [comment]
https://lite.3kh0.net/ (Games)

**Reply by DJshelfmushroom** [reply]
ok i did it

**Comment by DJshelfmushroom** [comment]


**Reply by YesTheProxy** [reply]
YOU HEARD ME

**Reply by theshmunguss1** [reply]
@YesTheProxy You need to stop deleting job applications

**Reply by YesTheProxy** [reply]
I don't
I haven't even been on the applications in weeks

**Reply by theshmunguss1** [reply]
NVM, forms bug

**Reply by YesTheProxy** [reply]
lol

**Reply by DJshelfmushroom** [reply]
you want me to incorporate these into the site? I can do this

**Reply by YesTheProxy** [reply]
Yup
